modal/cli/run.py

```diff
@@ -368,8 +368,9 @@ def get_command(self, ctx, func_ref):
 @click.option("-d", "--detach", is_flag=True, help="Don't stop the app if the local process dies or disconnects.")
 @click.option("-i", "--interactive", is_flag=True, help="Run the app in interactive mode.")
 @click.option("-e", "--env", help=ENV_OPTION_HELP, default=None)
+@click.option("-m", "--module", is_flag=True, help="use library module as a script")
```

Comment:
It migh be helpful to align this explanation with the terminology that we use in the [guide page](https://modal.com/docs/guide/mounting#absolute-vs-relative-imports) on the two modes of App reference. In particular, we call the _other_ mode "Script mode" so it's a little confusing to say that this flag invokes the module "as a script".

Also, nit: can we match the capitalization of the other help entries?